nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -98,12 +98,9 @@ let
   addDeviceNames =
     imap1 (idx: drive: drive // { device = driveDeviceName idx; });
 
-  efiPrefix =
-    if pkgs.stdenv.hostPlatform.isx86 then "${pkgs.OVMF.fd}/FV/OVMF"
-    else if pkgs.stdenv.isAarch64 then "${pkgs.OVMF.fd}/FV/AAVMF"
-    else throw "No EFI firmware available for platform";
-  efiFirmware = "${efiPrefix}_CODE.fd";
-  efiVarsDefault = "${efiPrefix}_VARS.fd";
+  efiPrefix = pkgs.OVMF.getEfiPrefix pkgs.stdenv.hostPlatform;
+  efiFirmware = efiPrefix.firmware;
+  efiVarsDefault = efiPrefix.variables;
```

Comment:
We might as well inline efiFirmware and efiVarsDefault to reduce the layers of redirection.

Reply:
You're right

Reply:
Done.